bin/setup

```diff
@@ -23,18 +25,15 @@ is_darwin (){
 
 has_homebrew (){
   echo 'Checking for homebrew'
-  [ is_darwin ] && [ hash brew >/dev/null 2>&1 ]
+  is_darwin && hash brew >/dev/null 2>&1
```

Comment:
* `[ darwin ]` is equivalent to `test "darwin"` (`[` is an alias to `test`) which always evaluates to true, because "darwin" is a non-empty string.
* `[ hash ... ]` is passing multiple arguments to `[` which it does not take.

Fixed these here and elsewhere in the file to match the intention of what this script was trying to do.